docs/area-owners.md

```diff
@@ -93,7 +93,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-System.Globalization                      | @ericstj      | @dotnet/area-system-globalization                   |                                                                                                                                                                                                                                                                                           |
 | area-System.IO                                 | @jeffhandley  | @dotnet/area-system-io                              |                                                                                                                                                                                                                                                                                           |
 | area-System.IO.Compression                     | @jeffhandley  | @dotnet/area-system-io-compression                  | <ul><li>Also includes System.IO.Packaging</li></ul>                                                                                                                                                                                                                                       |
-| area-System.IO.Pipelines                       | @kevinpi      | @davidfowl @halter73 @jkotalik                      |                                                                                                                                                                                                                                                                                           |
+| area-System.IO.Pipelines                       | @ericstj      | @davidfowl @halter73 @jkotalik                      |                                                                                                                                                                                                                                                                                           |
```

Comment:
Should this be @ericstj or @danmoseley?

Reply:
@rafikiassumani-msft perhaps you?

Reply:
Should be @adityamandaleeka, we look at new pipelines issues in runtime triage

Reply:
sounds good